assignment-1-sql/chimi/chimi-sherpa.sql

```diff
@@ -0,0 +1,183 @@
+-- Week 2 Assignment --
+-- QUESTION 1 ---------------------------------------------
+
+SELECT *
+FROM fact_order
+WHERE charge_amount > 500
+AND completed_at is NOT null
+
+ORDER BY completed_at desc
+LIMIT 1
+
+
+-- Question 2 --------------------------------------------
+
+SELECT 
+    DATE_TRUNC('week', fo.completed_at) as period,
+
+COUNT(DISTINCT fo.id) AS order_volume,
+SUM(fo.total_charged_amt) AS GMV
+
+FROM fact_order fo 
+JOIN zones z ON z.id = fo.zone_id
+JOIN regions r ON r.id = z.region_id
+
+WHERE
+r.country_id = 124
+AND fo.completed_at >= date_trunc('week', dateadd('week',-8,current_date()))
+AND fo.completed_at < date_trunc('week', current_date())
+AND fo.completed_at is not null
+
+GROUP BY 1
+ORDER BY period desc
+
+
+--Question 3 ----------------------------------------------
+
+with active_drivers as (
```

Comment:
https://blazer.instacart.tools/blazer/queries/136704-shoppers-grouped-by-retention

Another option where you do the heavy lifting of grouping first.